### PR TITLE
feat(form_customization): add separators

### PR DESCRIPTION
**Pull request by jbarreau**

### CODE REVIEW

File: src/datasource_toolkit/forestadmin/datasource_toolkit/interfaces/actions.py [hunk 1]
```diff
@@ -87,9 +89,19 @@ class ActionFieldType(enum.Enum):
     "UserDropdown",
 ]
 
+LayoutWidgetTypes = Literal["Separator", "Input"]
```
Comment (EnkiP) [inline src/datasource_toolkit/forestadmin/datasource_toolkit/interfaces/actions.py:92]: LayoutElement ? LayoutComponent ?

File: src/datasource_toolkit/forestadmin/datasource_toolkit/interfaces/actions.py [hunk 2]
```diff
@@ -87,9 +89,19 @@ class ActionFieldType(enum.Enum):
...
 
-class ActionField(TypedDict):
+
+class BaseActionFormElement(TypedDict):
     type: ActionFieldType
+
+
+class ActionLayoutElement(BaseActionFormElement):
+    component: LayoutWidgetTypes
+    fieldName: NotRequired[str]
```
Comment (EnkiP) [inline src/datasource_toolkit/forestadmin/datasource_toolkit/interfaces/actions.py:101]: fieldName => fieldId
and I don't think it should be in ActionLayoutElement. It should belongs to ActionLayoutInputElement

**Comment by EnkiP** [comment]
check ForestValueConverter.make_form_data, it seems like it tries to get form data from FormElements instead of fields

**Review by matthv — APPROVED** [review]
LGTM

**Pull request merged**